commercetools-java-client-apache-async/pom.xml

```diff
@@ -40,6 +40,37 @@
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
```

Comment:
May be we could pull this configuration into our parent pom and we should check that we don't release our `test-jar`s to maven central.